code/game/mob/living/carbon/human/human_defense.dm

```diff
@@ -612,12 +626,8 @@ bullet_act
 	if (hit_zone == "l_hand" || hit_zone == "r_hand" || hit_zone == "r_foot" || hit_zone == "l_foot")
 		if (prob(50))
 			if (prob(65))
-				if (user != src)
-					user.visible_message("<span class='danger'>[user] has tried to strike [src]'s [affecting.name] with \the [I.name] but missed!</span>", "<span class='danger'>You tried to strike [src]'s [affecting.name] with \the [I.name] but missed!</span>")
-					return
-				else
-					user.visible_message("<span class = 'danger'>[user] has tried to strike \his [affecting.name] with \the [I.name] but missed!</span>", "<span class = 'danger'>You tried to strike yourself with \the [I.name] but missed!</span>")
-					return
+				visible_message("<span class='notice'>[user] has tried to strike [src]'s [affecting.name] with [I.name] but missed!</span>")
```

Comment:
This changes the functionality and now everybody sees the same message

Reply:
redid the entire thing